gse_Inventory_Reports/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
can you review the name of the files and folder.
gse_inventory_reports and not gse_inventory_reports